style.css

```diff
@@ -247,3 +254,15 @@ th {
 td:first-child {
   color: #bbbbbb;
 }
+
+.gray {
+  color: #888888;
+}
+
+.red {
+  color: #ff0000;
+}
+
+.hidden {
+    display: none;
+}
```

Comment:
Now that we have this, can you apply it to the `type="file"` element from `index.html` and remove the rule about hiding these from above?